cpp/daal/src/algorithms/svm/svm_train_result.h

```diff
@@ -66,7 +66,7 @@ class SaveResultTask
         size_t nSV = 0;
         for (size_t i = 0; i < _nVectors; ++i)
         {
-            if (_alpha[i] > zero) nSV++;
+            if (_alpha[i] != zero) nSV++;
```

Comment:
Are variables (in the condition) floating point? Is it safe to compare by == or != for floating point in this context?

Reply:
Applied - replaced on `isZero`